src/openrct2-ui/windows/Map.cpp

```diff
@@ -1404,18 +1410,49 @@ class MapWindow final : public Window
             widgets[WIDX_MAP].bottom = height - 1 - 14;
     }
 
+    void CalculateTextLayout()
+    {
+        utf8 buffer[512];
+        Formatter ft;
+
+        int32_t textOffset = 4 + LIST_ROW_HEIGHT;
+        _firstColumnWidth = 118;
+        for (uint32_t i = 0; i < 4; i++)
+        {
+            OpenRCT2::FormatStringLegacy(buffer, sizeof(buffer), MapLabels[i], ft.Data());
```

Comment:
Formatter is empty here? what is this even doing. All you are doing here is getting the string for the maplabel. LanguageGetString(MapLabel[i]) would do that much simpler

```cpp
int32_t maxFirstColumnString = 0;
for (uint32_t i = 0; i < 4; i++)
{
    const auto label = MapLabels[i];
    const auto* labelStr = GetLanguageString(label);
    maxFirstColumnString = std::max(GfxGetStringWidth(labelStr, FontStyle::Medium), maxFirstColumnString);
}
_firstColumnWidth = std::max(textOffset + maxFirstColumnString, 118); // Is 118 defined anywhere??
```

Reply:
The last argument to `FormatStringLegacy` is a buffer, so it’s hardly surprising `ft` is empty.

Reply:
Does it make a difference if the map label has formatting tags in it?

Reply:
If they had formatting tags that need args then yes if they are just post process tags then no.  I guess should check which sort they are before making my sort of change.

Reply:
They're just colours. Seems to work fine with your suggestion. Thanks.